boundary-validation-for-dop/Readme.md

```diff
@@ -0,0 +1,16 @@
+# Service Description:
+
+The service supports creating user records.
```

Comment:
I would rename this directory as "data-validation"

Reply:
Done with https://github.com/ballerina-guides/integration-samples/pull/28/commits/95e981d60f608cc016e76bfcc86fd5769c091174